english/sc2/sc2.md

```diff
@@ -0,0 +1 @@
+It's 10 p.m. and you're borging, and decide to surf on google, for stacraft 1, an old game that you play some years ago, suddenly you read starcraft 2 a new version with beter graphics and you give it a chance again visiting [Starcraft 2](www.starcraft2.com)
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.